### ISSUE TITLE
Directional Light's direction isn't really clear

### ISSUE THREAD

**Issue by Athomield** [godotengine/godot#12616]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
WIN 10 godot 3 alpha 2

**Issue description:**

So what I did is create a simple prism and just play around with the meshs params. Now what I don't get here is when the direction of the light is looking at that faces normal, the face is dark (when it should be light) but when the direction of of the light is parallel to the face, the face is light (when it should be dark) (keep in mind i'm talking about the face making the angle and NOT the flat one)
Is it just a bug ? or am I being an idiot here ? Is this intended behavior ? (if it is intended behavior than it is a bit confusing, coming from unity) 

![2017-11-03 1](https://user-images.githubusercontent.com/7095418/32394473-897d97fc-c0dd-11e7-9516-21d025fd88b2.png)

![2017-11-03 2](https://user-images.githubusercontent.com/7095418/32394474-899f15e4-c0dd-11e7-804d-5c833b627062.png)

**Comment by raymoo** [comment]
I think it's a bug

**Comment by bojidar-bg** [comment]
Seems like the arrow is perpendicular from the actual direction, cc @djrm.

**Comment by djrm** [comment]
I dont have this problem, i've seen it before, but now i cant reproduce it, also no completely sure this has something to do with the directional light gizmo points.

**Comment by Athomield** [comment]
@drjm It's my stupid mistake to forget to mention that the actual project was created in godot 3.0 alpha 1 not 2 but I forgot what was in the scene when I was using that version (i think it had only a directional light) then I used a godot 3 30 October 2017 daily build to open the same project. So when I opened the godot  30 October 2017 daily build i created the MeshInstance with a prism on it and tried changing the subdivision values from the prism. When I tried changing the depth subdivision value for the first few times, the editor crashed .I was able to reproduce that once again, but not for a third time so I couldn't write a report about it.
